src/components/calculator/calculator.js

```diff
@@ -1,33 +1,48 @@
-        <button type="button">8</button>
-        <button type="button">9</button>
-        <button type="button" className="multply orange">X</button>
-        <button type="button">4</button>
-        <button type="button">5</button>
-        <button type="button">6</button>
-        <button type="button" className="sub orange">-</button>
-        <button type="button">1</button>
-        <button type="button">2</button>
-        <button type="button">3</button>
-        <button type="button" className="plus orange">+</button>
-        <button type="button" className="Zero">0</button>
-        <button type="button">.</button>
-        <button type="button" className="isqual orange">=</button>
-      </div>
-    </center>
-  </>
-);
+const MyCalculator = () => {
+  const [calculatorData, setCalculatorData] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const myValueButton = (buttonName) => {
+    const newData = calculate(calculatorData, buttonName);
+    setCalculatorData(newData);
+  };
+
+  const inputValue = calculatorData.next || calculatorData.total || '0';
+  return (
+    <>
+      <center>
+        <div className="container">
+          <input type="number" value={inputValue} id="calculator-input" readOnly />
+          <button type="button" onClick={() => myValueButton('AC')}>AC</button>
+          <button type="button" onClick={() => myValueButton('+')}>+/-</button>
+          <button type="button" onClick={() => myValueButton('%')}>%</button>
+          <button type="button" className="divide orange" onClick={() => myValueButton('÷')}>÷</button>
+          <button type="button" onClick={() => myValueButton('7')}>7</button>
+          <button type="button" onClick={() => myValueButton('8')}>8</button>
+          <button type="button" onClick={() => myValueButton('9')}>9</button>
+          <button type="button" className="multply orange" onClick={() => myValueButton('x')}>x</button>
+          <button type="button" onClick={() => myValueButton('4')}>4</button>
+          <button type="button" onClick={() => myValueButton('5')}>5</button>
+          <button type="button" onClick={() => myValueButton('6')}>6</button>
+          <button type="button" className="sub orange" onClick={() => myValueButton('-')}>-</button>
+          <button type="button" onClick={() => myValueButton('1')}>1</button>
+          <button type="button" onClick={() => myValueButton('2')}>2</button>
+          <button type="button" onClick={() => myValueButton('3')}>3</button>
+          <button type="button" className="plus orange" onClick={() => myValueButton('+')}>+</button>
+          <button type="button" className="Zero" onClick={() => myValueButton('0')}>0</button>
+          <button type="button" onClick={() => myValueButton('.')}>.</button>
+          <button type="button" className="isqual orange" onClick={() => myValueButton('=')}>=</button>
+        </div>
+      </center>
+    </>
+  );
+};
```

Comment:
- [ ] `README.md` file: I think it would be better if you update your `## Getting started` section by adding all information needed in order to setup your project locally. Right now, I noticed that it misses one important command of how to access the cloned directory before starting to install dependencies, and others. To help you I suggest you refer this **[template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**.

- [ ] I also suggest you update your README file by making it correctly matches the **[required template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**. You should ensure that all sections are ordered as from the template, as right now they are not ordered like on the **[template](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**. e.g.: `Future Features` section should come after `## Authors` section, `## Getting Started` should come before `## Author` sections,...

Reply:
Can be improved upon!